src/javascript/app/pages/user/new_account/virtual_acc_opening.js

```diff
@@ -86,6 +87,8 @@ const VirtualAccOpening = (() => {
             { selector: '#email_consent' },
             { request_field: 'utm_source',          value: TrafficSource.getSource(utm_data) },
             { request_field: 'new_account_virtual', value: 1 },
+            { request_field: 'signup_device',       value: signup_device },
+            { request_field: 'date_first_contact',  value: date_first_contact },
```

Comment:
I assume the declaration for these values needs to be before this? e.g. lines 102-104 need to be moved to  line 81